### PR TITLE
174 2 remove relative imports from macro files in digitalmarketplace govuk frontend

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/aYlQvPAJ/174-2-remove-relative-imports-from-macro-files-in-digitalmarketplace-govuk-frontend

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7739,25 +7739,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
+          "resolved": false,
```
Comment (katstevens) [inline package-lock.json:7742]: Can we leave this `package-lock` stuff out of this commit/PR? Apparently it's a bug in NPM

**Comment by lfdebrux** [comment]
Can we add a note to the changelog explaining the change? Something like

'- You no longer need to use the govuk-frontend macros bundled with digitalmarketplace-govuk-frontend; Digital Marketplace components will now use macros under the `govuk/` prefix in the template loader.'

File: package-lock.json (hunk 1 above)
Comment (agalamatis) [reply package-lock.json:7742]: Yes, I will take it out.

**Comment by lfdebrux** [comment]
I just realised that the ticket doesn't say anything about the relative imports in the Sass files, but we should probably change those as well. Are you okay to do that in this PR @agalamatis?

**Comment by agalamatis** [comment]
> I just realised that the ticket doesn't say anything about the relative imports in the Sass files, but we should probably change those as well. Are you okay to do that in this PR @agalamatis?

Looking into them now. The rest are done.

**Comment by lfdebrux** [comment]
Just tried this locally, it looks like with the changes to the Sass files this is a breaking change, apps will need to add `node_modules/digitalmarketplace-govuk-frontend` to the `includePaths` for node Sass. We should add that info to the changelog.

**Review by lfdebrux — APPROVED** [review]
Looks good to me. I've tweaked the changelog slightly also.

**Pull request merged**